grammar/basic-grammar.xml

```diff
@@ -365,6 +365,14 @@
     <opt>&WS;<non-terminal ref="Where"/></opt>
   </production>
 
+  <production name="LegacyFilterExpression" rr:inline="true" oc:legacy="true">
+    <seq>FILTER  &WS; ( &WS; <non-terminal ref="FilterExpression"/> &WS; ) </seq>
+  </production>
+
+  <production name="LegacyExtractExpression" rr:inline="true" oc:legacy="true">
+    <seq>EXTRACT &WS; ( &WS; <non-terminal ref="FilterExpression"/> &WS; <opt>&WS; | &expr;</opt> ) </seq>
+  </production>
```

Comment:
How about we group these two under the same production? They are extremely similar (as we can see by them being replaced by the same, more general thing). We could call that production f.e. `LegacyListExpression` or `LegacyFilterExpression` and then we can move things there that become deprecated. WDYT?

Reply:
It makes sense, I changed the rule to `LegacyListExpression`.